src/main/java/gameframework/gui/GameStatusBar.java

```diff
@@ -8,10 +8,22 @@
 
 import javax.swing.JPanel;
 
+/**
+ * This class represents the status of the bar contained into the window
```

Comment:
This comment doesn't help… in fact it contradicts the name of the class: /the status of the bar/ would be a class named `GameBarStatus`